schemas/yaml/Lexeme.yml

```
@@ -37,6 +37,17 @@ properties:
     pattern: '^[^ ]+$'
     description: A human-readable key that uniquely identifies this lexeme or variant within the language. Best practice is for the key to consist of a representation of the lemma form of the word without diacritics, and, if the word is a homonym, the homonym number. However, any value is acceptable as long as it is unique for the language. (Keys do not need to be unique across languages.)
 
+  alternativeAnalysis:
```

This should be plural since it's an array.

```suggestion
  alternativeAnalyses:
```